web/src/pages/people/index.tsx

```diff
@@ -26,12 +64,33 @@ export default function PeoplePage() {
         description={
           isAdmin
             ? "View and manage portal users, including adding via invitation or upload"
-            : isIAO
-              ? "View users in your projects or invite a collaborator"
-              : "You do not have permission to view this page"
+            : isTreOpsStaff
+              ? "View approved researchers"
+              : isIAO
+                ? "View users in your projects or invite a collaborator"
+                : "You do not have permission to view this page"
         }
       />
-      {isAdmin ? <AdminView /> : isIAO && <IAOView />}
+
+      {!isTreOpsStaff && (
+        <div className={styles["button-container"]}>
+          {isAdmin && <ApprovedResearcherImport />}
+          {(isAdmin || isIAO) && <ExternalInvite />}
+        </div>
+      )}
+      {!isAdmin && <Callout construction />}
+      {!users || users.length === 0 ? (
+        <Box>
+          <div className={styles["no-users-found"]}>No users found</div>
+          {errorMessage && (
+            <Alert type="error">
+              <AlertMessage>{errorMessage}</AlertMessage>
+            </Alert>
+          )}
+        </Box>
+      ) : (
+        <UserDataTable canEdit={true} users={users} setUsers={setUsers} isLoading={isLoading} />
+      )}
```

Comment:
Is there a benefit to passing `isLoading` to the child component here? You could equally render the loading message in this file and save rendering a component when it is not ready yet.